src/game/judgments.js

```diff
@@ -1,16 +1,25 @@
-
 import _ from 'lodash'
 
 export const UNJUDGED = 0
 export const MISSED = -1
 
 //#region judgment timegate
-export const JUDGMENTS = [
-  { value: 1, timegate: 0.020, endTimegate: 0.040 },
-  { value: 2, timegate: 0.050, endTimegate: 0.100 },
-  { value: 3, timegate: 0.100, endTimegate: 0.200 },
-  { value: 4, timegate: 0.200, endTimegate: 0.200 },
-]
+export const NORMAL_JUDGE = {
```

Comment:
Instead of exporting judges, export a function, e.g. `getJudgeForNotechart` which will pick an appropriate judge for the notechart. This way, the judgment logic with regards to notechart is also contained in this file.